opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-csv-domain.ts

```diff
@@ -65,8 +57,19 @@ export const defaultIngestionGroupsCount = async (context: AuthContext) => {
   return defaultGroupLength.length ?? 0;
 };
 
-export const userAlreadyExists = async (context: AuthContext, args: QueryUserAlreadyExistsArgs) => {
-  const users = await findAll(context, context.user, args) as UserConnection;
+export const userAlreadyExists = async (context: AuthContext, name: string) => {
+  // We use SYSTEM_USER because manage ingestion should be enough to create an ingestion Feed
+  const users: BasicStoreCommon[] = await findAll(context, SYSTEM_USER, {
```

Comment:
Could you rename to =>
```suggestion
  const users: BasicStoreCommon[] = await findAllUser(context, SYSTEM_USER, {
```